sys-libs/libatomic_ops/libatomic_ops-7.6.2.recipe

```diff
@@ -0,0 +1,65 @@
+SUMMARY="An atomic memory update operations portable implementation"
+DESCRIPTION="Semi-portable access to hardware-provided atomic memory update operations on a number architectures."
+HOMEPAGE="https://github.com/ivmai/libatomic_ops"
+COPYRIGHT="2003-2011 Hewlett-Packard Development Company, L.P."
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/ivmai/libatomic_ops/releases/download/v$portVersion/libatomic_ops-$portVersion.tar.gz"
+CHECKSUM_SHA256="219724edad3d580d4d37b22e1d7cb52f0006d282d26a9b8681b560a625142ee6"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	libatomic_ops$secondaryArchSuffix = $portVersion
+	lib:libatomic_ops$secondaryArchSuffix = 2.1.1 compat >= 1
+	lib:libatomic_ops_gpl$secondaryArchSuffix = 2.1.2 compat >= 1
```

Comment:
```diff
-	lib:libatomic_ops$secondaryArchSuffix = 2.1.1 compat >= 1
+	lib:libatomic_ops$secondaryArchSuffix = 1.1.1 compat >= 1
-	lib:libatomic_ops_gpl$secondaryArchSuffix = 2.1.2 compat >= 1
+	lib:libatomic_ops_gpl$secondaryArchSuffix = 1.1.2 compat >= 1
```

Reply:
Shouldn't we use [`libVersionCompat`](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#using-the-libversioncompat-user-variable)?